hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/domain/RecordItemBuilder.java

```diff
@@ -141,7 +150,11 @@ public Builder<ContractCallTransactionBody.Builder> contractCall(ContractID cont
 
         return new Builder<>(TransactionType.CONTRACTCALL, transactionBody)
                 .receipt(r -> r.setContractID(contractId))
-                .record(r -> r.setContractCallResult(contractFunctionResult(contractId)));
+                .record(r -> r.setContractCallResult(contractFunctionResult(contractId)))
+                .sidecarRecord(r -> {
+                    r.get(0).setStateChanges(contractStateChanges(contractId));
+                    r.get(1).setActions(contractActions(contractId));
```

Comment:
Should just add to the sidecar list instead of assuming a certain sized list is already present.

```suggestion
                .sidecarRecord(contractStateChanges(contractId))
                .sidecarRecord(contractActions(contractId));
```

Reply:
That makes sense. Thanks, I have changed it to add to the sidecar list rather than editing a default list.